projects/experimental/components/pdf-viewer/pdf-viewer.style.less

```diff
@@ -0,0 +1,80 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+tui-pdf-viewer {
+    display: block;
+    inline-size: 100%;
+    block-size: 100vh;
+    box-sizing: border-box;
+    color: var(--tui-text-primary);
+    padding-block-start: env(safe-area-inset-top);
+    padding-block-end: env(safe-area-inset-bottom);
+
+    --t-header: 4rem;
+
+    tui-root:not(._mobile) & {
+        background: var(--tui-background-accent-on-light, rgba(0, 0, 0, 1));
+    }
+
+    tui-root._mobile tui-dialog & {
+        --t-header: 2.5rem;
+
+        tui-app-bar[data-size] {
```

Comment:
Specificity here is already higher than on line 29, do we need [data-size]? Do we need it on line 29?